src/webpack/index.ts

```diff
@@ -97,12 +98,15 @@ export async function getConfig(): Promise<Configuration> {
     chunkFilename: 'static/[id]-[chunkhash].css'
   })
 
+  const bundleAnalyzerPlugin = getNeedAnalyze() ? new BundleAnalyzerPlugin() : null
```

Comment:
没必要塞一个 `null` 进去？建议是

```suggestion
if (getNeedAnalyze()) {
  config = appendPlugins(new BundleAnalyzerPlugin())
}
```

Reply:
done